test/Libraries/RevitIntegrationTests/WorkflowTests.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using Revit.Elements;
+using RevitTestServices;
+using Dynamo.Models;
+using RTF.Framework;
+using Autodesk.DesignScript.Geometry;
+using RevitServices.Persistence;
+using System.Collections.Generic;
+using RevitServices.Transactions;
+
+namespace RevitSystemTests
+{
+    [TestFixture]
+    public class WorkflowTests : RevitSystemTestBase
+    {
+        //this method will be deleted on Monday after Mr.Ritesh add it to RevitSystemTestBase
+        protected static IList<Autodesk.Revit.DB.Wall> GetAllWalls()
```

Comment:
Zora, As discussed, we need to move this method to RevitSystemTestBase class. I have added here to show you how can you access Walls from Revit.